Exception when using `lein-cprint` on Java >= 9.

Whenever I run `lein cprint` when using Java >= 9, I get the following error:

```
clojure.lang.Compiler$CompilerException: Syntax error compiling at (fipp/ednize.clj:1:1).
#:clojure.error{:phase :compile-syntax-check, :line 1, :column 1, :source "fipp/ednize.clj"}
 at clojure.lang.Compiler.load (Compiler.java:7652)
    clojure.lang.RT.loadResourceScript (RT.java:381)
    clojure.lang.RT.loadResourceScript (RT.java:372)
    clojure.lang.RT.load (RT.java:459)
    clojure.lang.RT.load (RT.java:424)
    clojure.core$load$fn__6856.invoke (core.clj:6115)
    clojure.core$load.invokeStatic (core.clj:6114)
    clojure.core$load.doInvoke (core.clj:6098)
    ...
    clojure.main$main_opt.invokeStatic (main.clj:514)
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
Caused by: java.lang.ExceptionInInitializerError: null
 at java.lang.Class.forName0 (Class.java:-2)
    java.lang.Class.forName (Class.java:398)
    clojure.lang.RT.classForName (RT.java:2212)
    clojure.lang.RT.classForName (RT.java:2221)
    clojure.lang.RT.loadClassForName (RT.java:2240)
    ...
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
Caused by: java.lang.ClassNotFoundException: java/sql/Timestamp
 at java.lang.Class.forName0 (Class.java:-2)
    java.lang.Class.forName (Class.java:398)
    clojure.lang.RT.classForName (RT.java:2212)
    clojure.lang.RT.classForNameNonLoading (RT.java:2225)
    clojure.instant/loading__6737__auto__ (instant.clj:9)
    clojure.instant__init.load (:9)
    ...
```

I'm since tracked this down to an issue in [`fipp`](https://github.com/brandonbloom/fipp/pull/78) which is a dependency of [`puget`](https://github.com/greglook/puget). I'm about to open a PR in `puget` to upgrade `fipp` after which `lein-cprint` will need to upgrade `puget` to resolve the issue.

--- a/project.clj
+++ b/project.clj
@@ -8,4 +8,4 @@
   :eval-in-leiningen true
 
   :dependencies
-  [[mvxcvi/puget "1.2.1"]])
+  [[mvxcvi/puget "1.3.2"]])

